packages/binding-opcua/test/client-test.ts

```diff
@@ -51,6 +48,10 @@ describe("OPCUA Client", function () {
     after(async () => {
         await client.stop();
     });
+    // server shall stop after client
```

Comment:
it is important to shutdown client before we shutdown server to prevent client complaining that server is not reachable and start a reconnection loop.

Reply:
this will get rid of this warning:
```
    ✔ Y4 - invokeResource (273ms)
10:32:19.837Z :client_base_impl              :523   attempt to recreate a new secure channel : suspended becasue reconnection is canceled !
10:32:19.840Z :client_base_impl              :1593  _recreate_secure_channel has failed: err =  Reconnection has been canceled - ClientSession
```